rskj-core/src/main/java/co/rsk/rpc/netty/JsonRpcCustomServer.java

```diff
@@ -68,14 +72,24 @@ protected JsonResponse handleJsonNodeRequest(final JsonNode node) throws JsonPar
             response = super.handleJsonNodeRequest(node);
             ExecTimeoutContext.checkIfExpired();
 
-            return response;
+        } else {
```

Comment:
Why did we move this here inside an `else` block?

Reply:
In order to measure the size of the object I need to get the response and calculate it. 
So if I understood it correctly, the condition in the line 71 `timeout <= 0` is the one used to decide what to do, if create the new timeout or not, so I added the if else to generate the response and the timeout and them calculate the size after this. 

The time to calculate the size is not being considered for these timeouts but I consider it irrelevant. 

In any case the timeout is not working as expected for batch request so It will be refactored.